WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -274,22 +283,44 @@ final class MySiteViewController: UIViewController, UIScrollViewDelegate, NoSite
 
         // Set the nav bar
         navigationController?.navigationBar.accessibilityIdentifier = "my-site-navigation-bar"
+
+        if isSidebarModeEnabled {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(title: nil, image: UIImage(systemName: "ellipsis"), target: nil, action: nil, menu: UIMenu(options: .displayInline, children: [
+                UIDeferredMenuElement.uncached { [weak self] in
+                    $0(self?.sitePickerViewController?.makeSiteActionsMenu()?.children ?? [])
+                }
+            ]))
+        }
     }
 
     private func configureNavBarAppearance(animated: Bool) {
         if scrollView.contentOffset.y >= 60 {
             if isNavigationBarHidden {
-                navigationController?.setNavigationBarHidden(false, animated: animated)
+                setNavigationBarHidden(false, animated: animated)
             }
             isNavigationBarHidden = false
         } else {
             if !isNavigationBarHidden {
-                navigationController?.setNavigationBarHidden(true, animated: animated)
+                setNavigationBarHidden(true, animated: animated)
             }
             isNavigationBarHidden = true
         }
     }
 
+    private func setNavigationBarHidden(_ isHidden: Bool, animated: Bool) {
+        if isSidebarModeEnabled {
+            navigationItem.titleView = isHidden ? UIView() : {
+                let button = UIButton.makeMenuButton(title: navigationItem.title ?? Strings.mySite)
+                button.addAction(UIAction { [weak self] _ in
+                    self?.sitePickerViewController?.siteSwitcherTapped()
+                }, for: .touchUpInside)
```

Comment:
Nitpick: using `.primaryActionTriggered` instead of `.touchUpInside`?

Reply:
Updated. I totally missed when it was added.